### PR TITLE
Tweak bug report template

### PR DESCRIPTION
**Pull request by fabpot**
| Q             | A
| ------------- | ---
| Branch?       | 6.1
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | n/a 
| License       | MIT
| Doc PR        | n/a

Let's make it clear that without a way to reproduce a bug, there is little chance we will be able to help or fix.

/cc @xabbuh as you are doing that manually a lot, let's try to fix the root cause ;)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_Bug_report.md [hunk 1]
```diff
@@ -11,12 +11,15 @@ labels: Bug
 <!-- A clear and concise description of the problem. -->
 
 **How to reproduce**
-<!-- Code and/or config needed to reproduce the problem. If it's a complex bug,
-     create a "bug reproducer" as explained in:
+<!-- \!/ This is the most important part of the report. \!/
```
Comment (nicolas-grekas) [inline .github/ISSUE_TEMPLATE/1_Bug_report.md]: Let's use ⚠️?

File: .github/ISSUE_TEMPLATE/1_Bug_report.md [hunk 2]
```diff
@@ -11,12 +11,15 @@ labels: Bug
...
+     Without a way to easily reproduce your issue, there is little chance we will be able to help you and work on a fix.
+     Please, take the time to show us some code and/or config that is needed to reproduce the problem.
```
Comment (nicolas-grekas) [inline .github/ISSUE_TEMPLATE/1_Bug_report.md]: that is needed *for others* to reproduce

File: .github/ISSUE_TEMPLATE/1_Bug_report.md (hunk 1 above)
Comment (fabpot) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: done

File: .github/ISSUE_TEMPLATE/1_Bug_report.md (hunk 2 above)
Comment (fabpot) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: done

File: .github/ISSUE_TEMPLATE/1_Bug_report.md [hunk 3]
```diff
@@ -11,12 +11,15 @@ labels: Bug
 <!-- A clear and concise description of the problem. -->
 
 **How to reproduce**
-<!-- Code and/or config needed to reproduce the problem. If it's a complex bug,
-     create a "bug reproducer" as explained in:
+<!--  ⚠️  This is the most important part of the report ⚠️
+     Without a way to easily reproduce your issue, there is little chance we will be able to help you and work on a fix.
+     Please, take the time to show us some code and/or config that is needed for others to reproduce the problem.
+     If it's a complex bug, create a "bug reproducer" as explained in:
```
Comment (xabbuh) [inline .github/ISSUE_TEMPLATE/1_Bug_report.md]: Maybe we should suggest to create a bug reproducer by default? I often see people posting code fragments which would be enough to reproduce something, but still requires someone to try it out to copy and paste these snippets into an application. That's something that takes time and we could make it more clear IMO that making it as easy as possible to reproduce an issue increases the likelihood for someone to look into it.

Though I have no idea how to phrase that in a short way.

**Comment by stof** [comment]
Maybe we can turn this into an issue form rather than a markdown template, which allows us to put more visible help texts on each field (and mark some as required), instead of having HTML comments in the markdown.

**Comment by fabpot** [comment]
@stof Done, not sure if we need to keep both the md and the yaml files.

File: .github/ISSUE_TEMPLATE/1_Bug_report.md (hunk 3 above)
Comment (fabpot) [reply .github/ISSUE_TEMPLATE/1_Bug_report.md]: @xabbuh I've tweaked the text, how does that sound to you?

**Comment by stof** [comment]
@fabpot no. GitHub will ignore the md file if it has a yaml template for the same name

**Comment by stof** [comment]
shouldn't this be done in the 4.4 branch ? IIRC, we have managed those templates in the oldest maintained branch and merged them up until now (including the recent change done by @nicolas-grekas)

**Comment by fabpot** [comment]
Rebased on 4.4, where the md files do not exist.
For some reasons, they exist on 5.3, they are removed here #44420

**Review by nicolas-grekas — APPROVED** [review]

**Comment by fabpot** [comment]
FYI, I've resync all Github issue templates from 4.4 to 6.1 so that in the future, we can always merge in the lowest maintained branch like for translations (even if Github will only always use the main branch version).

**Pull request merged**